test/androidTest/java/org/thoughtcrime/securesms/NicknameEspressoTest.java

```diff
@@ -62,7 +62,7 @@ public void canSetGroupNickname() {
             .goConversation()
             .goConversations()
                 .assertText(testString)
-            .goGroup(testString)
+            .goGroup()
```

Comment:
as I said, I have overloaded that method, did you continue working from my ui fixes ??

Reply:
yep and I removed the overload

the tests should not care what group they join. this is also the pattern used for the conversation, the number does not need to be specified.

Reply:
but then you will endup creating 100s of groups, the reason I overloaded is to check if I renamed group TESTTESTTEST to "sdghajkhdfgdg", then in the same test i used goGroup("sdghajkhdfgdg") and then will open that specific test and reset it name to TESTTESTTEST. now my cellphone I have 10 new random groups, whereas in the past I had only TESTTESTTEST. I really recommend/suggest use goGroup("new nickname") to reset it and make it ready for the next ui test

Reply:
there is a nick name ui test that also relays on "TESTTESTTEST" group. if the test does not find that group it will create new one. We don't want that. just simply after you rename a group, in the same test, go back to the conversation group, access that newly named group and reset it to TESTTESTTEST.

Reply:
it still does all of that, but just doesn't allow the caller to specify the test. you got spammed with groups cause I kept deleting them while testing this code.

Reply:
good justification